core/Session/SessionFingerprint.php

```diff
@@ -105,4 +105,9 @@ public function isRemembered()
         $userInfo = $this->getUserInfo();
         return !empty($userInfo['remembered']);
     }
+
+    public function updateSessionStartTime()
+    {
+        $_SESSION[self::SESSION_INFO_SESSION_VAR_NAME]['ts'] = Date::now()->getTimestampUTC();
```

Comment:
do I see this right that the timestamp was basically unused before that?

Reply:
if I understand your question correctly, it's used via `getSessionStartTime()`, which is used in SessionAuth::isSessionStartedBeforePasswordChange(). Which I think is actually an issue... might need to explicitly store the expiration time instead of updating the start time.

Reply:
that's why I'm asking :)